demos/ws-bootable-jar-example/pom.xml

```diff
@@ -0,0 +1,340 @@
+                    <plugins>
+                        <plugin>
+                            <groupId>org.wildfly.plugins</groupId>
+                            <artifactId>wildfly-jar-maven-plugin</artifactId>
+                            <!-- Bare metal but we still need to ensure that artifacts are installed
+                                 correctly into maven repo -->
+                            <executions>
+                                <execution>
+                                    <!-- Create bootable jar for bare metal environment - use already
+                                         defined configuration and add filename & cloud support -->
+                                    <id>bootable-baremetal</id>
+                                    <configuration>
+                                        <output-file-name>${project.build.finalName}-bootable-baremetal.jar</output-file-name>
+                                    </configuration>
+                                    <phase>process-test-sources</phase>
+                                    <goals>
+                                        <goal>package</goal>
+                                    </goals>
+                                </execution>
+                            </executions>
+                        </plugin>
+                        <plugin>
+                            <!-- Install artifacts created by wildfly-jar-maven-plugin plugin
+                                 manually since the plugin installs it under same name -->
+                            <groupId>org.apache.maven.plugins</groupId>
+                            <artifactId>maven-install-plugin</artifactId>
+                            <executions>
+                                <execution>
+                                    <id>install-bootable-jar-baremetal</id>
+                                    <phase>generate-test-resources</phase>
+                                    <goals>
+                                        <goal>install-file</goal>
+                                    </goals>
+                                    <configuration>
+                                        <groupId>${project.groupId}</groupId>
+                                        <artifactId>${project.artifactId}</artifactId>
+                                        <version>${project.version}</version>
+                                        <packaging>jar</packaging>
+                                        <classifier>bootable-baremetal</classifier>
+                                        <file>${project.build.directory}/${project.build.finalName}-bootable-baremetal.jar
+                                        </file>
+                                    </configuration>
+                                </execution>
+                            </executions>
+                        </plugin>
+                    </plugins>
+                </pluginManagement>
+            </build>
+        </profile>
```

Comment:
Let's remove this block, as we're specifically targeting OpenShift here, and not building for bare-metal will trim the example build duration to 1/2.

Reply:
done